src/screens/CreatePost/elements/SheetAddTopic.js

```diff
@@ -6,14 +6,13 @@ import {ScrollView, StyleSheet, Text, TextInput, TouchableNativeFeedback, View}
 import Card from './Card';
 import TopicItem from '../../../components/TopicItem';
 import {Button} from '../../../components/Button';
+import {colors} from '../../../utils/colors';
 import {convertString} from '../../../utils/string/StringUtils';
 import {fonts} from '../../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for colors has been updated, but the `useDynamicColors` hook and its usage have been removed. If this hook was used to support dynamic theme changes (like light/dark mode), this change might break that functionality. Please ensure that the new color handling approach still supports all required features.

```diff
- import {COLORS} from '../../../utils/theme';
- import {useDynamicColors} from '../../../hooks/useToggleColors';
+ import {colors} from '../../../utils/colors';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->